### ISSUE TITLE
yathesis v1.0.2 and biblatex "heading" \printbibliography argument

### ISSUE THREAD

**Issue by degeuser** [dbitouze/yathesis#74]
With yathesis v1.0.2, when using biblatex \printbibliography with heading=none argument (or actually any other heading or title argument), a Bibliography title is still used in addition to that given as an optional argument (see attached image).

This issue does not happen with v0.99.

Any idea on where it may come from ?

![image](https://user-images.githubusercontent.com/20084227/83158587-3d164700-a105-11ea-9564-44d0f802d544.png)

**Comment by dbitouze** [comment]
Could you please post a [MCE](https://texnique.fr/osqa/faq/#custom-id-ecm) (ECM in French)?

**Comment by degeuser** [comment]
Thanks for your prompt reply.
Here is a MCE as requested.
```
\documentclass{yathesis}

\usepackage{biblatex}
\addbibresource{\jobname.bib}
\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@book{keya,
  author = {Author, A.},
  year = {2001},
  title = {TitleA},
  publisher = {Publisher},
}

@book{keyb,
  author = {Author, B.},
  year = {2001},
  title = {TitleB},
  publisher = {Publisher},
}
\end{filecontents}

\begin{document}

\chapter{Liste des publications}

Un peu de texte

\nocite{*}

\printbibliography[heading=subbibnumbered,title={Articles dans des journaux à comité de lecture}]

\end{document}
```
A capture of the output on my computer is shown below (note the "Bibliographie" before the title)
![image](https://user-images.githubusercontent.com/20084227/83173614-7062d100-a119-11ea-9481-bb4f60f01482.png)

**Comment by degeuser** [comment]
Simply replacing yathesis class by book class in the previous MCE gives 
![image](https://user-images.githubusercontent.com/20084227/83243955-f5dc9480-a19e-11ea-9f9c-103f38ad9fc2.png)
Note the absence of "Bibliographie"

**Comment by dbitouze** [comment]
It is indeed a bug. I'm working on it. Meanwhile, you could try to add the following in your preamble:
```latex
\makeatletter
\AtEndPreamble{%
  \renewcommand{\printbibliography}[1][]{%
    \booltrue{YAD@printbibliography@used}%
    \singlespace%
    \YAD@printbibliography@ORI[heading=none,#1]%
    \endsinglespace%
  }%
}%
\makeatother
```

**Comment by degeuser** [comment]
Thank you very much!! This seems to do the trick !

**Comment by dbitouze** [comment]
Issue fixed with the latest version (1.0.3) of `yathesis`, provided by both TeX Live and MiKTeX.
